src/backend/services/carValueService.ts

```diff
@@ -0,0 +1,43 @@
+interface CarValue {
+  id: number;
+  model: string;
+  year: number;
+}
+
+let carValues: CarValue[] = [];
+
+export const getCarValues = (): CarValue[] => {
+  return carValues;
+};
+
+export const createCarValue = (model: string, year: number): CarValue => {
+  const newCarValue: CarValue = {
+    id: carValues.length + 1,
+    model,
+    year,
+  };
+
+  carValues.push(newCarValue);
+
+  return newCarValue;
+};
+
+export const getCarValueById = (carValueId: number): CarValue | undefined => {
+  return carValues.find((t) => t.id === carValueId);
+};
+
+export const calculateCarValue = (model: string, year: number): number => {
+  const alphabet = "abcdefghijklmnopqrstuvwxyz";
+  let carValue = 0;
```

Comment:
you could declare this variable globally so when returned the other functions could access the car's value. Then maybe use it in your array creation